packages/react-component-library/src/components/RangeSlider/Slider.tsx

```diff
@@ -128,7 +128,7 @@ export const RangeSlider: React.FC<RangeSliderProps> = ({
             </div>
           )}
         </Tracks>
-        {step && (
+        {tickCount && (
```

Comment:
Wondering why there isn't more accompanying this change? Like a test?

Reply:
> Looks good 👍
> 
> Some of the commits felt difficult to follow and wondering if some of them belonged as a single commit.

That's probably because of the way I was stepping through the bullet points in this issue: https://github.com/Royal-Navy/design-system/issues/1318